packages/web/src/store/errors/reportToSentry.ts

```diff
@@ -27,8 +27,8 @@ const jsLoggerMapping: { [level in ErrorLevel]: ConsoleLoggingMethod } = {
   Log: 'log'
 }
 
-const isResponseError = (error: Error): error is ResponseError =>
-  'response' in error && error.response instanceof Response
+export const isResponseError = (error: unknown): error is ResponseError =>
+  error instanceof Error && error.name === 'ResponseError'
```

Comment:
Responded about this in Slack, but I'm wondering if we shouldn't just update our compilation target? `instanceof` is a pretty useful/common pattern for checking things and since we don't have access to the internals of third-party code, it would be nice to maintain the ability to use it.